governance/polls/JAT1 to JAT2 Asset Reallocation - September 4, 2023.md

```diff
@@ -0,0 +1,61 @@
+---
+title: JAT1 to JAT2 Asset Reallocation - September 4, 2023
+summary: Signal your support or opposition for reallocating USD 500M from JAT1's ETF holdings to the T-bills ladder under JAT2.
+discussion_link: https://forum.makerdao.com/t/proposal-to-revise-asset-allocation-of-jat1-and-jat2/21718
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-09-04T16:00:00
+end_date: 2023-09-07T16:00:00
+---
+# Poll: JAT1 to JAT2 Asset Reallocation - September 4, 2023
+
+The Reserve Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 4 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following JAT1 (James Asset Trust 1) to JAT2 (James Asset Trust 2) Asset Reallocation:
+* Selling the ETF products within JAT1.
+* Paying any proceeds from this sale after expenses in excess of 500 million USD into the surplus buffer.
+* Deploying the 500 million USD towards the T-bills ladder already installed under JAT2.
+
+To execute this asset reallocation, the following practical steps will be undertaken:
+1. The Trustee will initiate a complete liquidation of all ETF holdings in the JAT1 portfolio.
+2. Any proceeds from the ETF liquidation that exceed USD 500M, after accounting for any related expenses, will be converted to DAI. These DAI funds will then be paid into the MakerDAO surplus buffer.
+3. The Trustee will sign an agreement that effectively transfers the remaining USD 500M from the JAT1 cash assets into JAT2.
+4. Following the agreement, the Trustee will execute a transfer of the USD 500M from JAT1's cash account to JAT2's cash account.
+5. The Trustee will request that the newly transferred funds be added to the existing US Treasury 6 month ladder execution arrangement with Sygnum Bank.
+6. After all the funds have been moved from JAT1 to JAT2, the Trustee will proceed to formally dissolve JAT1 as it will no longer hold any assets.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/proposal-to-revise-asset-allocation-of-jat1-and-jat2/21718) to inform your position on this issue before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
```

Comment:
Does this change require an executive vote? Or will JAT1 and JAT2 be acting off the result of the poll?